### ISSUE TITLE
Reenable Wifi Resets

### ISSUE THREAD

**Issue by lbussy** [thorrak/tiltbridge#143]
Wifi reset functionality was disabled, needs to be reenabled.

**Comment by lbussy** [comment]
Reenabled with eSPI, also added a board reset on GPIO35 while I was at it.

Should be working with ssd1306 but there's an include error I've not had time to hunt down.

Not working on TFT, need to figure out how touch works with the new lib.

Working on this in `reset_buttons` for now.

**Comment by lbussy** [comment]
Reset and WiFi resets now working for ssd1306 and eSPI.

The ssd1306 has two buttons on the face which are supported by the case Donnie made.  One is the boot button (which is now the WiFi reset) and one is a real reset.  

The eSPI has buttons in the same basic place but one is GPIO0 (which is now WiFiReset) and one is GPIO35 which I have made into a software reset to match the external configuration and functionality of the ssd1306.

For some reason, the TFT is not registering screen taps, even after adding back in `squix78/ESP8266 and ESP32 OLED driver for SSD1306 displays` (which is the new name of the old `ESP8266_SSD1306` lib.)  I can validate with a test sketch, but n over-simplified in-situ test refuses to allow the taps to register.  I hated that functionality but I don't want a new release to break previously functional workflows.

Still working on this in `reset_buttons`.
